test/extensions/transport_sockets/tls/tls_throughput_benchmark.cc

```diff
@@ -33,6 +33,19 @@ static void appendSlice(Buffer::Instance& buffer, uint32_t size) {
   buffer.commit(&slice, 1);
 }
 
+static void addTenFullSizeSlices(Buffer::Instance& buffer) {
```

Comment:
The name isn't quite correct. What this actually does is simulate 10 full size reads from a socket.

Reply:
Attempted to address the naming issue.